common/static/js/vendor/pdfjs/viewer.html

```diff
@@ -108,9 +108,9 @@
               <span data-l10n-id="download_label">Download</span>
             </button>
 
-            <a href="#" id="secondaryViewBookmark" class="secondaryToolbarButton bookmark visibleSmallView" title="Current view (copy or open in new window)" tabindex="55" data-l10n-id="bookmark">
+            <button type="button" id="secondaryViewBookmark" class="secondaryToolbarButton bookmark visibleSmallView" title="Current view (copy or open in new window)" tabindex="55" data-l10n-id="bookmark">
```

Comment:
Are you saying do not fix the accessibility in this file? Or that you just don't prefer it? I'm all for leaving vendor files alone, but we use a fair number of them and:

* it's not ideal ignoring these, and 
* it's unlikely we'll be able to push a fix up-stream efficiently

Also, this has been lingering for three weeks so I'd like to get it in.

But if you don't want me editing vendor files, I'll back out this edit and ignore this particular part of the platform.